lib/l10n/app_de.arb

```diff
@@ -155,5 +155,14 @@
     "password": "Passwort",
     "questionaireDoctorVisit1": "Waren Sie ",
     "questionaireDoctorVisit2": " bei einem Arzt oder in einem Krankenhaus?",
-    "questionaireDoctorVisitUnscheduled": "ungeplant"
+    "questionaireDoctorVisitUnscheduled": "ungeplant",
+    "information5tmg": "Angaben gem. § 5 TMG",
+    "projectDISTANCE": "Das Projekt DISTANCE ist einer der vom Bundesministerium für Bildung und Forschung (BMBF) geförderten \"Digitalen FortschrittsHubs: Gesundheit im Rahmen der Medizin-Informatikinitiative\".",
```

Comment:
Is it correct that "FortschrittsHubs" is written together?

Reply:
I received this piece of text from other stakeholders and I suppose that it's written in the way above.